core/src/main/scala/org/bitcoins/core/api/ChainQueryApi.scala

```diff
@@ -39,5 +50,49 @@ object ChainQueryApi {
     override def getNumberOfConfirmations(
         blockHashOpt: DoubleSha256DigestBE): Future[Option[Int]] =
       FutureUtil.none
+
+    /** Returns the block height of the given block stamp */
+    override def getHeightByBlockStamp(blockStamp: BlockStamp): Future[Int] =
+      Future.successful(0)
```

Comment:
is it the best thing to do to return successful futures here rather than `failed` ones?

Reply:
Yes. This way we don't need to fix the unit tests